app/controllers/invitations_controller.rb

```diff
@@ -9,52 +9,23 @@ def new
   end
 
   def create
-    flash.clear
-    errors = []
-    successes = []
-    for email in params[:emails].split(",").map{ |email| email.strip.downcase } do
+    emails = params[:emails].split(',').map { |email| email.strip.downcase } 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.